apps/climatemappedafrica/src/components/DataVisualisationGuide/index.js

```diff
@@ -0,0 +1,99 @@
+import { RichTypography } from "@commons-ui/core";
+import { Box, Grid, useMediaQuery } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Card from "@/climatemappedafrica/components/Card";
+import Carousel from "@/climatemappedafrica/components/Carousel";
+import Section from "@/climatemappedafrica/components/Section";
+
+function DataVisualisationGuide({ title, items }) {
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
+
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <Box
+      sx={{
+        padding: `${theme.typography.pxToRem(40)} 0`,
+      }}
```

Comment:
You don't need `theme`.
```suggestion
  const isDesktop = useMediaQuery((theme) => theme.breakpoints.up("lg"));

  if (!items?.length) {
    return null;
  }
  return (
    <Box
      sx={(theme) => ({
        padding: `${theme.typography.pxToRem(40)} 0`,
      })}
```

I don't think we need `pxToRem` either... `px` is good enough these days.

Reply:
@kilemensi We need the `isDesktop` variable so as to control dot visibility in the `carousel`
```jsx
<Carousel showDots={!isDesktop}>
...
</carousel>
```

Reply:
Yes @kelvinkipruto , my comment was you don't need a standalone `theme` variable. You can compute isDesktop as shown in my comment.